filebeat/module/elasticsearch/audit/_meta/fields.yml

```diff
@@ -0,0 +1,40 @@
+- name: audit
+  type: group
+  description: >
+  fields:
+    - name: node_name
+      description: "The name of the node"
+      example: "v_VJhjV"
+      type: keyword
+    - name: layer
+      description: "The layer from which this event originated: rest, transport or ip_filter"
+      example: "rest"
+      type: keyword
+    - name: event_type
+      description: "The type of event that occurred: anonymous_access_denied, authentication_failed, access_denied, access_granted, connection_granted, connection_denied, tampered_request, run_as_granted, run_as_denied"
+      example: "access_granted"
+      type: keyword
+    - name: origin_type
+      description: "Where the request originated: rest (request originated from a REST API request), transport (request was received on the transport channel), local_node (the local node issued the request)"
+      example: "local_node"
+      type: keyword
+    - name: origin_address
```

Comment:
So this is the ip of the machine on which for example an Elasticsearch client is running which makes requests?

Reply:
Yes